game/config.py

```diff
@@ -1,3 +1,6 @@
 # Set to True to enable computing expensive debugging information. At the time of
 # writing this only controls computing the waypoint placement zones.
 ENABLE_EXPENSIVE_DEBUG_TOOLS = False
+
+SERVER_BIND_ADDRESS = "127.0.0.1"
+SERVER_PORT = 5000
```

Comment:
I didn't know the old system at all, will we have to make this a configurable item at some point?

Reply:
Yeah, definitely eventually. Right now it'd be worthless, and even if the UI could run in a browser, we'd need to work out multi-user crap...